rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -1026,57 +1026,62 @@ where
                     vec![this_canister_id, canisters.root.unwrap()],
                 )
                 .await
-                .map_err(|e| {
+                .map_err(|err| {
                     format!(
-                        "Unable to set Root as Governance canister controller: {}",
-                        e
+                        "Unable to set SNS-W and Root as Governance canister controller: {}",
+                        err
                     )
                 }),
-            // Set root as controller of Ledger.
+            // Set Root as controller of Ledger.
```

Comment:
This issue predates this PR, but...

These comments sound like X will be the EXCLUSIVE controller of Y. One way to avoid this confusion: make these comments say something like "Add X as a controller of Y.". This pretty clearly indicates that existing controllers are not removed (in particular, we remain a controller of Y).

Reply:
[Ditto](https://github.com/dfinity/ic/pull/3978/files/ea18d84c309ec1103961ee7e10a6f57d9200a664#r1959987559)